iris_grib/tests/integration/load_convert/test_load_hybrid_height.py

```diff
@@ -35,8 +35,28 @@ class TestRealData(tests.IrisGribTest):
     def test_load_hybrid_height(self):
         filepath = self.get_testdata_path('faked_sample_hh_grib_data.grib2')
         cube = load_cube(filepath, 'air_temperature')
+        # Check that it loads right, and creates a factory.
         self.assertIsInstance(cube.aux_factories[0], HybridHeightFactory)
 
+    def test_levels_values(self):
+        filepath = self.get_testdata_path('faked_sample_hh_grib_data.grib2')
+        cube = load_cube(filepath, 'air_temperature')
+
+        # check actual model level values.
+        self.assertArrayAllClose(cube.coord('model_level_number').points,
```

Comment:
And also I think it would be safer (very marginally), or maybe just cleaner if we use `arrayEquals` instead of `arrayAllClose` here.

Reply:
:+1: